riker/api.py

```diff
@@ -30,7 +30,7 @@
 from retry import synchronize
 
 import fabric
-fabric.state.output.everything = False
+fabric.state.output.everything = True
```

Comment:
Ah, I see. OK, that'll have to do for now until we can get `riker dokku` to work without it.